tools/webpack

```diff
@@ -18,12 +18,16 @@ os.chdir(settings.DEPLOY_ROOT)
 STATIC_PATH = 'static/'
 
 
-def run():
+def run(quiet):
     # type: () -> None
```

Comment:
This needs mypy annotation so the CI can pass: `# type(bool) -> None`

Reply:
Oh wow missed that. Thanks @priyankp10! Would've taken me a while to figure that out.